api/envoy/api/v2/endpoint/endpoint.proto

```diff
@@ -62,8 +62,9 @@ message LbEndpoint {
   // balancer to select endpoints in a cluster for a given request. The filter
   // name should be specified as *envoy.lb*. An example boolean key-value pair
   // is *canary*, providing the optional canary status of the upstream host.
-  // This may be matched against in a route's ForwardAction metadata_match field
-  // to subset the endpoints considered in cluster load balancing.
+  // This may be matched against in a route's
+  // :ref:`envoy_api_msg_route.RouteAction` metadata_match field to subset the
```

Comment:
Thanks. Minor nit: Can you make this:
```
:ref:`RouteAction <envoy_api_msg_route.RouteAction>`
```
? This prevents the internal protodoc names leaking into visible docs.